requirements.txt

```diff
@@ -8,6 +8,7 @@ Jinja2>=2.7.3
 Sphinx>=1.2.3
 docopt>=0.6.2
 simplejson>=3.8.1
+urllib3>=1.9
```

Comment:
I think that urlib3 is probably brought by another lib. I think about request.
And resquest is brought by tortilla as an example. So we just have to add our top level lib.